packages/fluentui/react-northstar/src/utils/factories.ts

```diff
@@ -234,3 +241,20 @@ export function createShorthand<P>({
 
   return null;
 }
+
+export const createShorthand = <P>(
+  Component: React.ElementType<P> & { shorthandConfig?: ShorthandConfig },
+  value?: ShorthandValue<P>,
+  options?: CreateShorthandOptions<P>,
+) => {
+  const { mappedProp = 'children', allowsJSX = true, mappedArrayProp = undefined } = Component.shorthandConfig || {};
```

Comment:
Why do we need `mappedArrayProp = undefined` ? Isn't it going to be `undefined` anyway in that case?